lib/signaling/v2/peerconnection.js

```diff
@@ -1717,6 +1717,15 @@ function updateEncodingParameters(pcv2) {
       params.encodings[0].networkPriority = 'high';
     }
 
+    if (!isFirefox && sender.track.kind === 'video') {
+      // NOTE(mpatwardhan): All the simulcast encodings in Safari have
+      // the same resolution. So, we make sure that the lower layers have
+      // lower resolution, as seen in Chrome.
+      params.encodings.forEach((encoding, i) => {
+        encoding.scaleResolutionDownBy = 1 << i;
```

Comment:
```suggestion
        encoding.scaleResolutionDownBy = 1 << (encodings.length - i - 1);
```
@makarandp0 The first layer is usually the lowest resolution layer (Chromium has this behavior).